elementary/monitor/api/models/schema.py

```diff
@@ -38,17 +37,17 @@ def format_normalized_full_path_sep(cls, normalized_full_path: str) -> str:
 
 # NormalizedArtifactSchema must be first in the inheritance order
 class NormalizedModelSchema(NormalizedArtifactSchema, ModelSchema):
-    artifact_type: str = Field("model", const=True)
+    artifact_type: str = Field("model", const=True)  # type: ignore  # noqa
```

Comment:
unfortunately Field in pydantic 2 doesn't have a "const" field and that causes mypy to error. ignoring until we migrate to pydantic 2 for real